### PR TITLE
Rename download buttons, change download icon to envelope

### PR DESCRIPTION
**Pull request by gingerbreadpup**
Suggestions welcome...

### CODE REVIEW

File: src/pages/Account.vue [hunk 1]
```diff
@@ -26,7 +26,7 @@
                                 <div> {{ payment.pended_on }}</div>
                                 <div> {{ payment.vouchers.length }}</div>
                                 <div class="amount">£{{ payment.vouchers.length }}</div>
-                                <div class="download"><i class="fa fa-download" aria-hidden="true" title="Download"></i></div>
+                                <div class="email"><i class="fa fa-envelope" aria-hidden="true" title="Send record to my email"></i></div>
```
Comment (charlesstrange2) [inline src/pages/Account.vue:29]: Title quite long?

**Review by charlesstrange2 — APPROVED** [review]
PR100 !
:birthday: 

Maybe consider shortening the title, but otherwise OK.

:shipit:

File: src/pages/Account.vue (hunk 1 above)
Comment (gingerbreadpup) [reply src/pages/Account.vue:29]: It's just the title that shows when you hover over the icon, so I think it's ok.

**Comment by charlesstrange2** [comment]
Carry on and merge then :-)

**Comment by charlesstrange2** [comment]
W00t!

**Pull request merged**